pkg/controller/lifecycle/actuator.go

```diff
@@ -461,6 +461,10 @@ func getSeedResources(oidcReplicas *int32, hibernated bool, namespace, genericKu
 							PeriodSeconds:       20,
 							FailureThreshold:    3,
 						},
+						Env: []corev1.EnvVar{
+							{Name: "GOMAXPROCS", Value: "1"},      // in sync with the CPU limits
+							{Name: "GOMEMLIMIT", Value: "920MiB"}, // roughly 90% of 1Gi
```

Comment:
I guess we can do that, although I see not much benefit of doing so since these values are hardcoded anyway. How about leaving a comment next to where limits are defined so that one remembers to keep them in sync?